infrastructure/templates/public-api/ci/jobs/deploy-api-docs.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     environment: ${{ parameters.environment }}
     variables:
       docsPath: $(Pipeline.Workspace)/MainBuild/public-api-docs
-      nodeVersion: '20.16.0'
+      nodeVersion: '20.19.2'
```

Comment:
I think we can test this one with a Public API deployment from this branch.  I'll kick one off.